src/Profiles.ts

```diff
@@ -141,113 +131,231 @@ export class Profiles {
+                    value: userName
+                };
+                userName = await vscode.window.showInputBox(options);
+                if (userName === undefined) {
+                    vscode.window.showInformationMessage(localize("createNewConnection.undefined.username",
+                        "Operation Cancelled"));
+                    return;
+                }
+                schemaValues[value] = userName;
+                break;
+            case "password":
+                options = {
+                    placeHolder: localize("createNewConnection.option.prompt.password.placeholder", "Optional: Password"),
+                    prompt: localize("createNewConnection.option.prompt.password",
+                    "Enter the password for the connection. Leave blank to not store."),
+                    password: true,
+                    value: passWord
+                };
+                passWord = await vscode.window.showInputBox(options);
+                if (passWord === undefined) {
+                    vscode.window.showInformationMessage(localize("createNewConnection.undefined.passWord",
+                        "Operation Cancelled"));
+                    return;
+                }
+                schemaValues[value] = passWord;
+                break;
+            case "rejectUnauthorized":
+                const quickPickOptions: vscode.QuickPickOptions = {
+                    placeHolder: localize("createNewConnection.option.prompt.ru.placeholder", "Reject Unauthorized Connections"),
+                    ignoreFocusOut: true,
+                    canPickMany: false
+                };
+                const selectRU = ["True - Reject connections with self-signed certificates",
+                    "False - Accept connections with self-signed certificates"];
+                const ruOptions = Array.from(selectRU);
+                const chosenRU = await vscode.window.showQuickPick(ruOptions, quickPickOptions);
+                if (chosenRU === ruOptions[0]) {
+                    rejectUnauthorize = true;
+                } else if (chosenRU === ruOptions[1]) {
+                    rejectUnauthorize = false;
+                } else {
+                    vscode.window.showInformationMessage(localize("createNewConnection.rejectUnauthorize","Operation Cancelled"));
+                    return undefined;
+                }
+                schemaValues[value] = rejectUnauthorize;
+                break;
+            default:
+                const description: string = schema[value].optionDefinition.description.toString();
+                switch (schema[value].type) {
+                    case "string":
+                        options = await this.optionsValue(value, schema);
+                        const profValue = await vscode.window.showInputBox(options);
+                        schemaValues[value] = profValue;
+                        break;
```

Comment:
> > Also would prefer a shorter text too. Does anyone have an opinion on shortening it to:
> > _The base path for your API mediation layer instance. See the [documentation](https://docs.zowe.org/stable/web_help/index.html?p=zowe_profiles_create_zftp-profile) for more info._
> 
> I agree with Katelyn on the shorter text. Looks confusing with that much text
> _The base path for your API mediation layer instance._ Would suffice

@Colin-Stone @katelynienaber It looks like the base path property is currently being handled by this `default` case, which means the description text is retrieved from the schema defined by Zowe CLI or the Zowe Explorer extender. So as-is, the text shortening would have to be done in those respective repositories where they define the profile schema, rather than by Zowe Explorer.
- Alternatively, a base path `switch` case could be added into Zowe Explorer, similar to what was done for common profile properties such as `host`, `username`, `password`, etc. - In that case, a shortened base path description could be hard-coded there.

(Currently, there are `switch` cases for `host`, `port`, `user`, `password`, and `rejectUnauthorized`. The `default` case is used when the profile type the user is creating contains additional properties.)

Reply:
Sorry, my mistake!

Reply:
@tjohnsonBCM @zFernand0 Do you think it would be possible to handle the basePath description in zowe/CLI ? Can we open an issue for this?

Reply:
The description text is being fetched from the z/OSMF profile meta definition in `~/.zowe/profiles/zosmf/zosmf_meta.yaml`. If you want it to be shortened, we could an open an issue requesting a change in [these lines of code in Zowe CLI where the description originates](https://github.com/zowe/zowe-cli/blob/59820d6cefed5916d4807de78b90c68f4e56e237/packages/zosmf/src/ZosmfSession.ts#L88-L90).